src/tab-manager/index.js

```diff
@@ -107,7 +107,7 @@ const addTabs = ipcSender => tabsMetadata => {
       tabPreferences.session.setSpellCheckerDictionaryDownloadURL('file:///home/user/00-MN/projects/manusa/electronim/dictionaries/');
     }
     const tab = new BrowserView({webPreferences: tabPreferences});
-    tab.setAutoResize({width: true, height: true});
+    tab.setAutoResize({width: false, horizontal: false, height: false, vertical: false});
```

Comment:
the test assertion needs to match this argument

Reply:
i did not understood that error

Reply:
i have seen it..let me retry it

Reply:
:+1:

Reply:
you need to modify the test I linked in the issue so that it matches your changed in the production code:
https://github.com/manusa/electronim/blob/6b8158e1a6c30acf4e5812c5eefe83b9f83e6d02/src/tab-manager/__tests__/index.test.js#L121

`toHaveBeenCalledWith({/*...*/})`should be replaced with.....

Reply:
yeah thanks, I corrected it.

Reply:
If you did, you might have forgotten to commit your changes, it's not visible in the PR:
https://github.com/manusa/electronim/pull/144/files

Reply:
i did , it shows me 3 commits.

Reply:
![Web capture_7-10-2022_105451_github com](https://user-images.githubusercontent.com/102249987/194502194-1623fa5e-5458-4586-ba77-d4d275e24028.jpeg)

Reply:
sorry for taking long on this small task, seems i have to learn git well.

Reply:
you need to change the **two** files.
Once changed: `git add .`, `git commit -m 'fix: disabled tab-manager auto-resize', `git push origin main`

Reply:
i just did it says "Everything up-to-date".

Reply:
done

Reply:
do you know how i can remove/delete this commit

  [Merge remote-tracking branch 'upstream/main'](https://github.com/manusa/electronim/pull/144/commits/abfe3bd2ad21f40ffdf4a9e004bdd952bb40d430)

Reply:
You can alter the git history as you wish, however, at this point it would be much easier for you to start from scratch.

Also, you need to make sure that you are actually changing **two** files.

Reply:
okay.i'll start from scratch again.